lib/Mojo/Server/Daemon.pm

```diff
@@ -35,6 +35,7 @@ sub run {
   my $loop = $self->ioloop;
   my $int  = $loop->recurring(1 => sub { });
   local $SIG{INT} = local $SIG{TERM} = sub { $loop->stop };
+  $self->app->app_starting;
```

Comment:
This should trigger twice now.

Reply:
It did, hence the PID protection in `Mojolicious`.
If I remove this line, then `morbo ./myapp.pl` is not triggering the event!

Reply:
Right, that's a different problem though. This is newly designed feature, so it really shouldn't use workarounds like that.